mllib/src/main/scala/org/apache/spark/ml/fpm/PrefixSpan.scala

```diff
@@ -35,7 +37,87 @@ import org.apache.spark.sql.types.{ArrayType, LongType, StructField, StructType}
  */
 @Since("2.4.0")
 @Experimental
-object PrefixSpan {
+final class PrefixSpan(@Since("2.4.0") override val uid: String) extends Params {
+
+  @Since("2.4.0")
+  def this() = this(Identifiable.randomUID("prefixSpan"))
+
+  /**
+   * the minimal support level of the sequential pattern, any pattern that
```

Comment:
Use uppercase for the first char:

"""
Param for the minimal support level (default: `0.1`).
Sequential patterns that appear more than (minSupport * size-of-the-dataset) times are identified as frequent sequential patterns.
"""